sdk/servicebus/Azure.Messaging.ServiceBus/tests/ServiceBusTestBase.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+
+namespace Azure.Messaging.ServiceBus.Tests
+{
+    public class ServiceBusTestBase
+    {
+        protected static string ConnString = Environment.GetEnvironmentVariable("SERVICE_BUS_CONN_STRING", EnvironmentVariableTarget.Machine);
+        protected static string TenantId = Environment.GetEnvironmentVariable("TENANT_ID", EnvironmentVariableTarget.Machine);
+        protected static string ClientId = Environment.GetEnvironmentVariable("CLIENT_ID", EnvironmentVariableTarget.Machine);
+        protected static string ClientSecret = Environment.GetEnvironmentVariable("CLIENT_SECRET", EnvironmentVariableTarget.Machine);
+        protected const string QueueName = "josh";
+        protected const string SessionQueueName = "joshsession";
+        protected const string TopicName = "joshtopic";
+        protected const string Endpoint = "jolovservicebus.servicebus.windows.net";
```

Comment:
Josh, joshsession, joshtopic.. :smiley:

Reply:
Will remove this once Jesse's stuff is in. Or actually can probably be done in Jesse's PR.